bundles/org.smarthomej.binding.amazonechocontrol/src/main/java/org/smarthomej/binding/amazonechocontrol/internal/Connection.java

```diff
@@ -1326,20 +1327,22 @@ public void playAmazonMusicPlayList(Device device, @Nullable String playListId)
 
     public void announcement(Device device, String speak, String bodyText, @Nullable String title,
             @Nullable Integer ttsVolume, @Nullable Integer standardVolume) {
-        String plainSpeak = speak.replaceAll("<.+?>", " ").replaceAll("\\s+", " ").trim();
-        String plainBody = bodyText.replaceAll("<.+?>", " ").replaceAll("\\s+", " ").trim();
-
-        if (plainSpeak.isEmpty() && plainBody.isEmpty()) {
-            // if there is neither a bodytext nor (except tags) a speaktext, we have nothing to announce
+        speak = speak.replaceAll("\\s+", " ").trim();
+        bodyText = bodyText.replaceAll("\\s+", " ").trim();
+        String plainSpeak = speak.replaceAll("<.+?>", " ").trim();
+        String plainBodyText = bodyText.replaceAll("<.+?>", " ").trim();
```

Comment:
```suggestion
        String trimmedSpeak = speak.replaceAll("\\s+", " ").trim(); 
        String plainSpeak = trimmedSpeak.replaceAll("<.+?>", " ").trim();
        String plainBodyText = bodyText.replaceAll("\\s+", " ").replaceAll("<.+?>", " ").trim();
```

Codstyle: Do not assign new values to parameters, consider them `final` (event if not explicitly declared). Use a local variable instead.

Reply:
done